README.md

```diff
@@ -207,7 +207,7 @@ The shared storage worklet invocation methods (`addModule`, `run`, and `selectUR
     *   via specifying an content attribute, e.g.: 
```

Comment:
I was actually talking about the content attribute, not the IDL attribute. Noting that it's a [boolean attribute](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes) means you don't have to specify whether the attribute's value matters.

Reply:
Ok, just fixed it. (meant to do a PR, but forgot to switch  to my newly created branch and committed the one-word change directly to main, sigh.). In any case, let me know if that's what you had in mind, thanks!